WorkloadManagementSystem/PilotAgent/PilotLogger/PilotLogger.py

```diff
@@ -0,0 +1,231 @@
+""" Pilot logger module for the remote loggin system
+    Only functions send and connect are depended on the client library used.
+    The current implementation uses stomp.
+"""
+
+__RCSID__ = "$Id$"
+
+import stomp
+import sys
+import Queue
+#Since PilotLogger can be used without DIRAC relaying only on PilotLoggerTools, we
+#check two possiblities:
```

Comment:
This is also used in unit tests, that I added for those classes. I didn't want to separately load Tools in UT when I test PilotLogger.